routes/deleteUser.py

```diff
@@ -15,14 +15,14 @@ def deleteUser(userName, direct):
                 f'select * from users where lower(userName) = "{userName.lower()}"'
             )
             user = cursor.fetchone()
+            if not user:
+                message("1", f'USER: "{userName}" NOT FOUND')
+                return redirect(f"/{direct}")
```

Comment:
if there is no use we won't be able to fetch anything from the db on lines 21-22, so the app will fail